gpgit.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why are you using a virtual environment for bash? I have not seen this before, except for python and electron. Why for bash?

Reply:
It's not a virtual environment, the env command helps to finds the command within your `$PATH`. So if `bash` exists within `/bin` it will use that, but on several (BSD) systems it's in `/usr/bin/` or even `/usr/local/bin` which won't work with a hardcoded `/bin/bash`

Reply:
The specific problem is that `/bin/bash` is often (at the very least on OS X) a very limited bash-like shell which does not support the features needed by your script